src/Components/Publishing/EditorialFeature/Components/Vanguard2019/Components/TableOfContents.tsx

```diff
@@ -1,25 +1,106 @@
 import { Box, Sans, Serif } from "@artsy/palette"
+import {
+  NewlyEstablished1,
+  NewlyEstablished2,
+  NewlyEstablished3,
+} from "Components/Publishing/EditorialFeature/Components/Vanguard2019/Blobs/NewlyEstablished"
 import { ArticleData } from "Components/Publishing/Typings"
 import { times } from "lodash"
 import React from "react"
+import styled from "styled-components"
+import { slugify } from "underscore.string"
```

Comment:
We're already using lodash; could this work? https://lodash.com/docs/4.17.15#kebabCase

Reply:
This is not a new library, we use it in a lot of spots in publishing -- ensures no special characters are included in addition to formatting.